packages/web/src/components/tipping/tip-audio/TipAudioModal.tsx

```diff
@@ -20,7 +20,8 @@ import { TipSent } from './TipSent'
 const messages = {
   sendATip: 'Send Tip',
   confirm: 'Confirm',
-  tipSent: 'Tip Sent'
+  tipSent: 'Tip Sent',
+  holdOn: '⚠️ Hold On a Moment'
```

Comment:
Do we usually rely on these kinds of characters? Wondering if we need a new icon or escaped version for this

Reply:
generally we do not but in this case it seems we are. i've seen other scenarios where we are too, in the rewards panels I believe.